plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/src/com/google/cloud/tools/eclipse/appengine/deploy/CleanupOldDeploysJobTest.java

```diff
@@ -61,11 +61,13 @@ public int compare(File o1, File o2) {
     }
   }
 
-  private void createTestDirectories(IPath tempDir, int count) throws InterruptedException {
+  private void createTestDirectories(IPath tempDirectory, int count) throws InterruptedException {
+    long now = System.currentTimeMillis();
     // most recent directory will be "1", oldest is "<count>"
     for (int i = count; i > 0; --i) {
-      assertTrue(new File(tempDir.append(Integer.toString(i)).toOSString()).mkdir());
-      Thread.sleep(1000L); // we need at least 1 second to have different modification times
+      File testDirectory = new File(tempDirectory.append(Integer.toString(i)).toOSString());
+      assertTrue(testDirectory.mkdir());
```

Comment:
It's a little strange that you're asserting the results of creating your test data, since this isn't itself what you're testing;  but I guess it's OK. Had to think for a minute to figure out what you were doing here.